src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -514,7 +514,7 @@ bool ONNXAbsOp::inferShapes() {
 bool ONNXAddOp::inferShapes() {
   if (!getOperand(0).getType().isa<RankedTensorType>() ||
       !getOperand(1).getType().isa<RankedTensorType>()) {
-    emitError("ONNXAddOp inferShapes failed");
+    emitError("Shape tensor(s) not ranked");
```

Comment:
Not exactly understanding the error message - "Shape tensor(s) not ranked". The shape tensor (which is always 1D) is always ranked (with rank=1).

Reply:
Fixed.